simulation/__main__.py

```diff
@@ -16,6 +16,9 @@ def convert_to_timedelta(minutes_string):
 
     parser = argparse.ArgumentParser()
 
+    parser.add_argument("household_distribution_file", type=pd.read_csv)
+    parser.add_argument("history_file", default="history.jsonl")
```

Comment:
What does it mean for a mandatory arg to have a default value?

Reply:
Good point, this doesn't make sense. I'll remove it.